internal/tools/terraform/plan.go

```diff
@@ -40,3 +42,28 @@ func (c *Cmd) Destroy() error {
 
 	return nil
 }
+
+// Output invokes terraform output and returns the value.
+func (c *Cmd) Output(variable string) (string, error) {
+	stdout, _, err := c.run(
```

Comment:
Was this a case where we could get terraform to output JSON and parse that for additional "safety"?